python/tests/unit/contents/test_chat_message_content.py

```diff
@@ -12,20 +12,20 @@
 
 
 def test_cmc():
-    message = ChatMessageContent(role="user", content="Hello, world!")
+    message = ChatMessageContent(role=AuthorRole.USER, content="Hello, world!")
     assert message.role == AuthorRole.USER
     assert message.content == "Hello, world!"
     assert len(message.items) == 1
 
 
 def test_cmc_str():
-    message = ChatMessageContent(role="user", content="Hello, world!")
+    message = ChatMessageContent(role=AuthorRole.USER, content="Hello, world!")
```

Comment:
Please keep this as a string and add a assert for the test, the return value from a LLM is a str as well!